server/requests/icecream-requests.rest

```diff
@@ -11,7 +11,11 @@ POST http://localhost:3003/api/icecreams HTTP/1.1
 content-type: application/json
 
 {
-    "name": "seventh",
+    "name": "jumbo staircase",
     "isUserGenerated": true,
-    "userId": "dgfbnogrb"
+    "userId": "leewiz"
 }
+
+###
+GET http://localhost:3003/api/icecreams/suggestion/3
```

Comment:
Forgot to mention - I was testing the endpoint yesterday and here had to change for `suggestions` to launch the request. But everything worked like a charm!

Reply:
thanks! this is now updated in the latest on this branch